### PR TITLE
fixed_decimal: clarify documentation for position sign in padding functions

### PR DESCRIPTION
**Pull request by kartva**
Example:
`pad_start` documentation states that "Negative numbers have no effect". This is ambiguous between the number being formatted and the position number passed as an argument.

Resolve this and other similar instances. Example: "Negative position numbers have no effect"

### CODE REVIEW

File: utils/fixed_decimal/src/decimal.rs [hunk 1]
```diff
@@ -1012,7 +1012,7 @@ impl FixedDecimal {
 
     /// Zero-pad the number on the left to a particular position.
     ///
-    /// Negative numbers have no effect.
+    /// Negative position numbers have no effect.
```
Comment (younies) [inline utils/fixed_decimal/src/decimal.rs:1015]:
```suggestion
/// The position of the negative sign for negative numbers has no effect.
```

File: utils/fixed_decimal/src/decimal.rs [hunk 2]
```diff
@@ -1012,7 +1012,7 @@ impl FixedDecimal {
...
     ///
     /// Also see [`FixedDecimal::set_max_position()`].
     ///
```
Comment (younies) [inline utils/fixed_decimal/src/decimal.rs:1018]: Add an example for the negative numbers

**Review by younies — CHANGES_REQUESTED** [review]
Good catch, just changed the comment slightly and also asking for adding tests for negative numbers.

File: utils/fixed_decimal/src/decimal.rs (hunk 1 above)
Comment (sffc) [reply utils/fixed_decimal/src/decimal.rs:1015]: I think this won't render the way you are thinking. `# NOTE` creates a heading, and the indentation creates a monospace code block. You could consider

```rust
/// NOTE: The position of the negative sign for negative numbers has no effect.
```

but I think what Kartavya proposed first is fine.

**Review by sffc — APPROVED** [review]
LGTM, adding a docs test might be helpful

File: utils/fixed_decimal/src/decimal.rs (hunk 1 above)
Comment (younies) [reply utils/fixed_decimal/src/decimal.rs:1015]: Sure,

Also, I think it is better to have something like this:
```rust
/// NOTE: Negative numbers as input has no effect.
```
